go/flowctl/cmd-api-build.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"net/url"
+	"path/filepath"
+	"strings"
+
+	"github.com/estuary/flow/go/bindings"
+	"github.com/estuary/flow/go/capture"
+	"github.com/estuary/flow/go/materialize"
+	"github.com/estuary/protocols/catalog"
+	pf "github.com/estuary/protocols/flow"
+	"github.com/fatih/color"
+	log "github.com/sirupsen/logrus"
+	mbp "go.gazette.dev/core/mainboilerplate"
+)
+
+type apiBuild struct {
+	BuildID     string                `long:"build-id" required:"true" description:"ID of this build"`
+	Directory   string                `long:"directory" default:"." description:"Build directory"`
+	FileRoot    string                `long:"fs-root" default:"/" description:"Filesystem root of fetched file:// resources"`
+	Network     string                `long:"network" default:"host" description:"The Docker network that connector containers are given access to."`
+	Source      string                `long:"source" required:"true" description:"Catalog source file or URL to build"`
+	SourceType  string                `long:"source-type" default:"catalog" choice:"catalog" choice:"jsonSchema"`
```

Comment:
nit: This one is missing a description. If that's intentional because it's internal only, then it might be best to hide this from the help output altogether.